### DISCUSSION TITLE
benefits from the Student Developer Pack are no longer available

### DISCUSSION THREAD

**Discussion by vklodik** [community/community#54403] [GitHub Education]
### Hi everyone,

Initially at the moment I was added to campus I had access to all the Student pack benefits, but now I have what is written below in the screenshot.
This is very strange because the PRO icon in my profile is still there, and I managed to access the Github Copilot, but I can't get courses from Educative now because the benefits are now unavailable 

![image](https://user-images.githubusercontent.com/37158403/236084686-938acccd-cb2c-429e-acbd-20267c5cc699.png)

**Comment by samus-aran** [comment]
Hey @vklodik  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!


I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
